Update Rust project READMEs (#1073)
dfinity/examples: rust/x509/README.md

## Prerequisites
- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/getting-started/install) v`0.23.0` or newer.
- [x] Clone the example dapp project: `git clone https://github.com/dfinity/examples`

> The minimum version of `dfx` thatrun this example locally is

Author: ```suggestion
```

> -   [x] Clone the example dapp project: `git clone https://github.com/dfinity/examples`

Author: ```suggestion

```

`root_ca_certificate` and `child_certificate` methods, those are rendered in the
web UI.

## Deploying the canister on the mainnet

> To deploy this canister the mainnet, you must do two things:

Author: ```suggestion
Before deploying this canister to the mainnet, you must do the following:
```
